recipes-support/opencv/opencv_4.4.0.imx.bb

```diff
@@ -0,0 +1,57 @@
+require recipes-support/opencv/opencv_4.4.0.bb
```

Comment:
We should not require existing recipes. Instead, we should include the original recipe content as is and do the required changes below it, marking it accordingly.

One example of this can be seen at:

https://github.com/Freescale/meta-freescale/blob/master/recipes-graphics/wayland/weston_9.0.0.imx.bb